fenix/app/src/test/java/org/mozilla/fenix/addons/AddonDetailsBindingDelegateTest.kt

```diff
@@ -85,7 +85,7 @@ class AddonDetailsBindingDelegateTest {
                 ratingUrl = "https://example.org/",
             ),
         )
-        assertEquals("100", binding.reviewCount.text)
+        assertEquals("100", binding.reviewCount.text.toString())
```

Comment:
Should we addition cases to verify we have the underline added ?

Reply:
as discussed on slack, I am not sure how we can cover that with an assertion :(